package-lock.json

```diff
@@ -1,18580 +1,8 @@
 {
   "name": "chartjs-plugin-annotation",
   "version": "1.0.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
very old npm?

Reply:
my fault. I'm gonna replace with the master version